cmd/notary-server/dev-config.json

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
We no longer need it. The only difference between configs was the TLS config. Now we have a way for everything to work out of the box for mac and linux, so we no longer need dev.